CreeDictionary/CreeDictionary/views.py

```diff
@@ -23,7 +23,9 @@ def lemma_details(request, lemma_text: str = None):  # pragma: no cover
     :param lemma_text: the exact form of the lemma (no spell relaxation)
     """
     extra_constraints = {
-        k: v for k, v in request.GET.items() if k in {"pos", "lc", "analysis", "id"}
+        k: v
+        for k, v in request.GET.items()
+        if k in {"pos", "full_lc", "analysis", "id"}
```

Comment:
This was responsible for the "nôhtêpayiw" case. Wordform objects generates constraints with `full_lc` rather than `lc`.

So to here's a simple explanation the bug: 

1. On the search page, one of the two _nôhtêpayiw_ lemma has `href=/word/nôhtêpayiw?full_lc=VII-3`
2. When the user clicks on the link, the `full_lc` is dropped here by this faulty code.
3. Without the `full_lc` constraint, the app finds out there are two potential matches, thus redirects the user to the same search page with the two results.To the user, it's as if the server isn't responding and that the entries do not have paradigms.

Reply:
Good detective work!